### PR TITLE
Adding ssh key usage

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Adding the usage of the public SSH key to the placeholder when creating a new profile.
### Changes

Updated the placeholder value in the public SSH key field.
### Related Issues

#896

### CODE REVIEW

File: src/elements/profiles/Profiles.wc.svelte [hunk 1]
```diff
@@ -68,7 +80,7 @@
     // ] },
     { label: "Mnemonics", symbol: "mnemonics", placeholder: "Enter Your Mnemonics", type: "password" },
     // { label: "TFChain Configurations Secret", symbol: "storeSecret", placeholder: "  Secret key used to encrypt your data on TFChain", type: "password" },
-    { label: "Public SSH Key", symbol: "sshKey", placeholder: "Your public SSH key will be added as default to all deployments.", type: "text" },
+    { label: "Public SSH Key", symbol: "sshKey", placeholder: "Disabling the ssh field means you won't be able to access your deployed solutions.", type: "text" },
```
Comment (AhmedHanafy725) [inline src/elements/profiles/Profiles.wc.svelte:83]: can we get the placeholder back to ssh usage, and put the current message as tooltip on the switch?

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]
the ssh field is not disabled after activating the profile
image

File: src/elements/profiles/Profiles.wc.svelte (hunk 1 above)
Comment (maayarosama) [reply src/elements/profiles/Profiles.wc.svelte]: Done.

**Comment by AhmedHanafy725** [comment]
> the ssh field is not disabled after activating the profile image

this is still not resolved.

**Comment by AhmedHanafy725** [comment]
the button hides the tooltip message, can it be above the button?
image

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**